src/test/lambda/wizards/samDeployWizard.test.ts

```diff
@@ -234,6 +239,7 @@ describe('SamDeployWizard', async () => {
                 promptUserForSamTemplate: async () => vscode.Uri.file(templatePath),
                 promptUserForRegion: async () => region,
                 promptUserForS3Bucket: async () => s3Bucket,
+                promptForNewS3BucketName: async () => undefined,
```

Comment:
No reason to deviate from the pattern followed by all the other symbols, which start with `promptuserFor` (though it's too verbose IMO, but arbitrary differences are worse than consistent verbosity)
```suggestion
                promptUserForNewS3BucketName: async () => undefined,
```

This is a general practice: if there is not a strong reason to introduce inconsistency, prefer consistency.

Reply:
Great advice, changed for consistency.